man/logind.conf.xml

```diff
@@ -297,6 +297,13 @@
       </varlistentry>
 
       <varlistentry>
+        <term><varname>InhibitorsMax=</varname></term>
+
+        <listitem><para>Controls the maximum number of concurrent inhibitors to permit. Defaults to 8192
+        (8K).</para></listitem>
+      </varlistentry>
```

Comment:
Yeah, so I thought about this, we set UserTasksMax to 12K after all in the same file, and RLIMIT_NOFILE defaults to 4K these days, which makes 96M, which is of course a tiny bit more than 8K... ;-)

But then, again, inhibitors are not too important currently, after all they little more than just a friendly request, and aren't honoured by privileged users anyway, so I figured this was actually OK for now.

The alternative would be to introduce UserInhibitorsMax= or so and enforce a per-user limit, but that would be quite some book-keeping in logind, and I was too lazy to add that, for this non-dangerous feature.

Or to say this differently: I think it is really important that we put limits on all our resources we open up to unprivileged code, simply so that whatever we allocate has a global limit. Whether users can fuck each other with that, I think is much less important (at least in this case). I think it's more important to enforce limits on what we allocate than it is to introduce a new DoS condition here.

Or to say it even differently: I think we need a global limit anyway. If one day somebody cares enough to add UserInhibitorsMax=, then that'd be great, but I think that can happen in a different PR.